code/datums/special_traits/traits/traits.dm

```diff
@@ -131,94 +147,97 @@
 /datum/special_trait/corn_fed
 	name = "Corn Fed"
 	greet_text = span_notice("My diet was quite rich in corn.")
-	weight = 100
+	weight = 150
 
 /datum/special_trait/corn_fed/on_apply(mob/living/carbon/human/character, silent)
-	character.change_stat("constitution", 2)
+	character.change_stat("constitution", 6)
 	character.change_stat("intelligence", -2)
 
 /datum/special_trait/bookworm
 	name = "Bookworm"
 	greet_text = span_notice("I'm a fan of books and I enjoy reading them regularly.")
-	weight = 100
+	weight = 0
```

Comment:
Ayo WHAT THE FUCK, I LOVE READING

Reply:
I could just make it so this also gives you intelligence stat. ATM, it doesn't do anything at all.